pyfar/signals/files.py

```diff
@@ -0,0 +1,515 @@
+
+    # load brir
+    guitar = pf.io.read_audio(os.path.join(file_dir, files[0]))
+    guitar.fft_norm = "rms"
+
+    # level to make sure all contents have approximately the same loudness
+    guitar.time *= .6
+
+    # resample brir
+    if sampling_rate != 48000:
+        guitar = pf.dsp.resample(guitar, sampling_rate)
+
+    return guitar
+
+
+def speech(voice="female", sampling_rate=44100):
+    """
+    Get anechoic speech sample.
+
+    The samples were taken from 'Music for Archimedes' [#]_ (Tracks 4, 5) with
+    kind permission of Bang & Olufsen for research and personal purposes. Any
+    commercial use or publication requires approval of Bang & Olufsen. For more
+    information on the recordings see [#]_.
+
+    .. note ::
+
+        **Copyright**: Bang & Olufsen
+
+    Parameters
+    ----------
+    voice : str, optional
+        Choose between a ``'female'`` (default) and ``'male'`` voice.
+    sampling_rate : int, optional
+        The sampling rate of the sample in Hz. The default of ``44100`` uses
+        the sample as it is, any other value uses
+        :py:func:`~pyfar.dsp.resample` for resampling to the desired
+        sampling rate.
+
+    Returns
+    -------
+    speech : Signal
+        The speech sample.
+
+    References
+    ----------
+    .. [#] Music for Archimedes. Bang & Olufsen, 1992, CD B&O 101.
+
+    .. [#] V. Hansen, and G. Munch, 'Making Recordings for Simulation Tests in
+           the Archimedes Project,' J. Audio Eng. Soc. 39, 768–774 (1991).
+    """
+
+    # download files if requires
+    files = _load_files('speech')
+
+    # load brir
+    file = files[0] if voice == "female" else files[1]
+    speech = pf.io.read_audio(os.path.join(file_dir, file))
+    speech.fft_norm = "rms"
+
+    # level to make sure all contents have approximately the same loudness
+    gain = 0.5 if voice == "female" else 0.3
+    speech.time *= gain
+
+    # resample brir
+    if sampling_rate != 44100:
+        speech = pf.dsp.resample(speech, sampling_rate)
+
+    return speech
+
+
+def brir(diffuse_field_compensation=False, sampling_rate=48000):
+    """
+    Get a binaural room impulse response (BRIR).
+
+    The BRIR was recorded with the FABIAN head and torso simulator in the
+    Berliner Philharmonie [#]_ (Emitter 17). The head of FABIAN was rotated
+    25 degree to the right. For more information see [#]_. A matching room
+    impulse response can be obtained by :py:func:`~rir`.
+
+    .. note ::
+
+        **License**: CC BY-NC-SA 4.0, David Ackermann, Audio Communication
+        Group, Technical University of Berlin
+
+    Parameters
+    ----------
+    diffuse_field_compensation : bool, optional
+        Apply a diffuse field compensation to the BRIR. This can be used as a
+        simple headphone compensation filter when listening to the BRIR. The
+        default is False, which does not apply the compensation. The diffuse
+        field compensation is taken from :py:func:`~hrirs`
```

Comment:
I think it's not really clear, that the compensation filter is part of the FABIAN database (I would have expected the inversion as part of the function here)

Reply:
We would need an entire spherical HRTF data set to calculate in on the fly. I thus used the pre-calculated.